compiler/passes/insertWideReferences.cpp

```diff
@@ -2368,7 +2364,6 @@ insertWideReferences(void) {
   FnSymbol* heapAllocateGlobals = heapAllocateGlobalsHead();
 
   if (!requireWideReferences()) {
-    convertNilToObject();
```

Comment:
Is this preemptive? (Prep for the nil changes?)

Reply:
Sorry this file is in this PR by mistake. I will remove these changes from this PR - they are from another PR. Thank you for drawing my attention to it!